Core/Debugger/DisassemblyManager.cpp

```diff
@@ -41,6 +42,39 @@ bool isInInterval(u32 start, u32 size, u32 value)
 	return start <= value && value <= (start+size-1);
 }
 
+bool IsLikelyStringAt(uint32_t addr) {
+	uint32_t maxLen = Memory::ValidSize(addr, 128);
+	if (maxLen <= 1)
+		return false;
+	const char *p = Memory::GetCharPointer(addr);
+	// If there's no terminator nearby, let's say no.
+	if (memchr(p, 0, maxLen) == nullptr)
+		return false;
```

Comment:
I guess you could also just cap and add "..." or something to the printed text - but I guess most interesting strings are likely to be shorter than 128 anyway.

Reply:
Right.  I was also worried about false positives.  I think if there's a stretch of 5000 AAAAs, it's probably not a string.

-[Unknown]

Reply:
If you wanna go really crazy you could dig for statistical properties of the data to see if text or not :)

(not serious heh)

Reply:
I almost checked if it was in the rodata section (that's why I even went down that path of suballocations.)  But decided it was more useful not checking that.

-[Unknown]